src/d/ir/instruction.d

```diff
@@ -12,10 +12,8 @@ private:
 	BasicBlock[] basicBlocks;
 
 public:
-	ref inout(BasicBlock) opIndex(BasicBlockRef i) inout in {
-		assert(i, "null block ref");
-		assert(i.index <= basicBlocks.length, "Out of bounds block ref");
-	} do {
+	ref inout(BasicBlock) opIndex(BasicBlockRef i) inout in(i, "null block ref")
+			in(i.index <= basicBlocks.length, "Out of bounds block ref") {
```

Comment:
This is not so good, they rest is great.

Reply:
let’s revert this one then